Content.Client/Atmos/UI/GasFilterBoundUserInterface.cs

```diff
@@ -58,9 +58,16 @@ private void OnFilterTransferRatePressed(string value)
 
         private void OnSelectGasPressed()
         {
-            if (_window is null || _window.SelectedGas is null) return;
-            if (!int.TryParse(_window.SelectedGas, out var gas)) return;
-            SendMessage(new GasFilterSelectGasMessage(gas));
+            if (_window is null) return;
+            if (_window.SelectedGas is null)
+            {
+                SendMessage(new GasFilterDeselectGasMessage());
+            }
```

Comment:
why does this need a separate network message? just make the `Gas` in `GasFilterSelectGasMessage` nullable